### PR TITLE
fix: Corrects missing dates in Network Upgrade cards

### PR DESCRIPTION
**Pull request by darigovresearch**
Would resolve https://github.com/ethereum-cat-herders/ech-website-v2/issues/124 if merged

### CODE REVIEW

File: network_upgrades/index.html [hunk 1]
```diff
@@ -369,7 +369,7 @@ <h3 class="post-title">
                               target="_blank" class="mr-10 black-color"><br>Execution Specs</a>
                            <a href="https://github.com/ethereum/consensus-specs/tree/dev/specs/bellatrix"
                               target="_blank" class="mr-10 black-color">Consensus Specs</a><br>
-                           <span class="post-date"><i class="fa fa-clock-o mt-48"></i> </span>
+                           <span class="post-date"><i class="fa fa-clock-o mt-48"></i>12/04/2023 </span>
```
Comment (poojaranjan) [inline network_upgrades/index.html:372]: 2022-09-15

File: network_upgrades/index.html [hunk 2]
```diff
@@ -340,7 +340,7 @@ <h3 class="post-title">
                               target="_blank" class="mr-10 black-color"><br>Execution Specs</a>
                            <a href="https://github.com/ethereum/consensus-specs/tree/dev/specs/capella"
                               target="_blank" class="mr-10 black-color">Consensus Specs</a><br>
-                           <span class="post-date"><i class="fa fa-clock-o mt-48"></i> </span>
+                           <span class="post-date"><i class="fa fa-clock-o mt-48"></i> 29/06/2022</span>
                            <a href="https://blog.ethereum.org/2023/03/28/shapella-mainnet-announcement" target="_blank"
```
Comment (poojaranjan) [inline network_upgrades/index.html:344]: 12/04/2023

**Review by poojaranjan — COMMENTED** [review]
Please update dates as in comment

**Comment by darigovresearch** [comment]
Thanks @poojaranjan for the reviews, both items should now be resolved.

Don't hesitate to reach out if you need anything else!

**Review by geovgy — APPROVED** [review]
Looks good now. Thanks for the PR.

**Pull request merged**